yoti_python_sandbox/version.py

```diff
@@ -1 +1 @@
-__version__ = "1.0.0"
+__version__ = "1.0.1"
```

Comment:
I think this might need to be `1.1.0` as it's adding new features

```suggestion
__version__ = "1.1.0"
```